### ISSUE TITLE
Filed to run cpd tasks

### ISSUE THREAD

**Issue by emartynov** [openbakery/gradle-xcodePlugin#203]
Good morning Rene,

Sorry, for report but looks like I'm testing already new functionality that is not available in 0.9.14.

I've tried to run cpd task and it failed with:
```
Can't get https://raw.githubusercontent.com/jkennedy1980/Objective-C-CPD-Language/master/releasesObjCLanguage-0.0.7-SNAPSHOT.jar to /Users/310198568/.gradle/caches/gxp/ObjCLanguage-0.0.7-SNAPSHOT.jar
```

The path should be corrected.

**Comment by renep** [comment]
It looks like the releasesObjCLanguage-0.0.7-SNAPSHOT.jar is gone, an I did not find any alternative. 

Therefor my solution would be do delete the CPD stuff from the plugin (for now) as long there is no objective-c support available: see https://pmd.github.io/pmd-5.3.3/pmd-objectivec/index.html

**Comment by emartynov** [comment]
It is there!

Just correct the path to:
```
https://raw.githubusercontent.com/jkennedy1980/Objective-C-CPD-Language/master/releases/ObjCLanguage-0.0.7-SNAPSHOT.jar
```

But it is quite old as well

**Comment by renep** [comment]
ok, was too blind ;)

**Comment by renep** [comment]
I have added OCLint support because it does more analytics. The CPD stuff was a pull request I have merged, and I it does not hurt as long the things works. Maybe it is going to be better in the future, because the CPD has move to github recently.

**Comment by emartynov** [comment]
I've tried to play with OCLint today and I'm a bit disappointed. Especially with defining all rules that I want

**Comment by renep** [comment]
fixed with 0.11.5
